### PR TITLE
should be everything

### PR DESCRIPTION
**Pull request by crodeheaver**

### CODE REVIEW

**Comment by crodeheaver** [comment]
@barberboy I did use body-parser for kicks and giggles, if that's ok.

**Comment by barberboy** [comment]
Yes, I'm so glad you did! Separate inputs is a _much_ better solution than multiple form elements. I had a mental lapse when I was explaining it in class, otherwise I would have recommended doing it that way. This looks great.

File: Counter.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (barberboy) [inline Counter.js]: Please export a `counter` function so we can create multiple instances of a counter.

If you don't want to use `.prototype`, feel free to define a function that returns a new instance using `Object.create(prototype)`.

**Comment by crodeheaver** [comment]
That should do it. I was originally having problems getting it to do what I wanted, so I gave up and did the object.

**Comment by keawade** [comment]
I forgot @barberboy didn't go over body-parser in class and went ahead and told everyone who came for tutoring to use it for their form submissions, haha. Pretty sure he won't mind too much.

Just couldn't see any reason why we would ever want separate routes and forms for what we are doing.

**Pull request closed without merging**